js/counter.js

```diff
@@ -54,10 +79,10 @@ function buyTeenagers(num) {
 
 function buyKettles(num) {
   kettleIncrease = () => {
-    if (PLAYER.points >= 50){
+    if (PLAYER.points >= 50 * num){
```

Comment:
Glad this got fixed, but as a bug-fix it should be in its own commit. Since it is unrelated to the feature "teenager prices are scaling".